webdriver-spec.html

```diff
@@ -3919,7 +3919,10 @@
  allow lookup of individual elements and collections of elements.
  Element retrieval searches are performed
  using pre-order traversal of the document’s nodes
- that match the provided selector’s expression.
+ that match the provided selector’s expression, and ignore the value
+ of the <a href="#dfn-page-load-strategy" class="internalDFN"
```

Comment:
Ignoring the value of the page load strategy makes no sense in this context. The page load strategy only applies when, well, actually loading a document.